### PR TITLE
sys-kernel/dracut-crypt-ssh: new package

### PR DESCRIPTION
**Pull request by cbcb**
Closes: https://bugs.gentoo.org/617436
Package-Manager: Portage-2.3.19, Repoman-2.3.6

### CODE REVIEW

**Comment by cbcb** [comment]
@gentoo-repo-qa-bot fixed the bugzilla link

File: sys-kernel/dracut-crypt-ssh/dracut-crypt-ssh-1.0.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="early unlocking of encrypted systems via ssh for dracut"
```
Comment (mgorny) [inline sys-kernel/dracut-crypt-ssh/dracut-crypt-ssh-1.0.6.ebuild]: Please fix the variable order and spacing to match the skel. Also, where did you get the template for this ebuild? I'm noticing a repeating misordering and I can't locate the source.

```
DESCRIPTION
HOMEPAGE
SRC_URI

LICENSE
SLOT
KEYWORDS
IUSE

DEPEND
RDEPEND
```

Comment (cbcb) [reply sys-kernel/dracut-crypt-ssh/dracut-crypt-ssh-1.0.6.ebuild]: I started with the skeleton that automatically appears in in vim.

I guess the LICENSE line wandered up after I corrected the license from "gpl-2" to "GPL-2" before I opened the pull request, so no common source for that error, I think.

**Comment by mgorny** [comment]
Thanks. Looks good, builds fine. I'm not going to test whether it works but I presume you've tested it yourself ;-).

**Pull request closed without merging**